README.md

```diff
@@ -1,76 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome Books
 
-> One paragraph statement about the project.
+> A web app to store books.
 
-![screenshot](./app_screenshot.png)
+The app allows you to add, store and remove books.
 
-Additional description about the project and its features.
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
+## Below is a GIF showing the app
+![gif](awesome-book.gif)
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
+You can see [the app live here](https://sahar-abdelsamad.github.io/Awesome-Books/)
 
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+## Built With
 
-### Prerequisites
+- HTML, CSS, JavaScript.
 
-### Setup
 
-### Install
+## To get a local copy up and running follow these simple example steps.
 
 ### Usage
-
-### Run tests
-
-### Deployment
-
-
+- Clone the repository using Git, Github Desktop or a version control manager of your choice.
+- Open index.html with a web browser.
```

Comment:
- [ ] **Required:** Include the setup header to write the steps another developer could follow to use your app. Eg. `git clone`, `npm install`, etc.

Reply:
Done :+1:  Thank you :smile: